collect_app/build.gradle

```diff
@@ -321,8 +313,6 @@ dependencies {
     testImplementation "org.powermock:powermock-module-junit4:2.0.5"
     testImplementation "org.powermock:powermock-api-mockito2:2.0.5"
 
-    testImplementation "androidx.arch.core:core-testing:2.1.0"
```

Comment:
I'm surprised we're not using this! This is a current dependency https://developer.android.com/jetpack/androidx/releases/arch. Fine to take it out if nothing is breaking I guess!